bin/yarn-install

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+require('shelljs/global')
+
+exec('cd cli && yarn && cd ..')
+exec('cd exo-add && yarn && cd ..')
+exec('cd exo-clone && yarn && cd ..')
```

Comment:
This `&&` stuff might not work on Windows. Please run each command individually. Also, ObservableProcess allows you to set the working directory in which you want to run the command, so you don't need all this `cd` madness here.